.travis.yml

```diff
@@ -1,17 +1,11 @@
 language: julia
-notifications:
-  email: peter.zion@gmail.com
+os:
+    - linux
+    - osx
 julia:
-    - 0.5
     - 0.6
-#    - nightly
-before_install:
-  - sudo apt-get update -qq -y
-  - sudo apt-get install libpcre3-dev -y
-  - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
-script:
-  - julia -e 'Pkg.init()'
-  - julia -e 'Pkg.clone(pwd()); Pkg.build("LibBSON")'
-  - julia -e 'Pkg.test("LibBSON", coverage=true)'
+notifications:
+    email: false
 after_success:
-- julia -e 'cd(Pkg.dir("LibBSON")); Pkg.add("Coverage"); using Coverage; Coveralls.submit(Coveralls.process_folder())'
+  - julia -e 'Pkg.add("Coverage"); cd(Pkg.dir("LibBSON")); using Coverage; Coveralls.submit(Coveralls.process_folder())'
```

Comment:
These can be combined:
```yaml
  - julia -e 'VERSION < v"0.7.0-DEV" || (using Pkg); cd(Pkg.dir("LibBSON")); Pkg.add("Coverage"); using Coverage; Coveralls.submit(Coveralls.process_folder()); Codecov.submit(Codecov.process_folder())'
```

Reply:
I usually stick with Codecov.
Do you have both of them set up on this repo?

Reply:
I'll have to check and make sure Codecov is added for both, and get the ids for the badges.
If you look at the README.md for the packages in juliastring.org, you'll see I like to be pretty complete
(but that is for later, not for this PR)

Reply:
I guess it won't need the part about using Pkg, since this is only for v0.6